Source/Scene/Cesium3DTile.js

```diff
@@ -120,6 +120,23 @@ define([
         this.children = [];
 
         /**
+         * Descendant tiles that need to be visible before this tile can refine. For example, if
+         * a child is empty (such as for accelerating culling), its descendants with content must
+         * be loaded first. If a tiles's children all have content, this is left undefined.
+         *
+         * @type {Array}
+         * @readonly
+         */
+        this.descendantsWithContent = undefined;
```

Comment:
I don't think any metadata would help right now, unless it could link to those descendants directly, which doesn't seem possible.